src/reactions/halloween.ts

```diff
@@ -0,0 +1,34 @@
+import { logger } from '../utilities/logger';
+import { ReactionDef } from './reaction-def';
+
+export const halloweenReaction: ReactionDef = {
+  emoji: '🎃',
+  description: 'Adds the halloween role to the user who reacts',
+  command: async (reaction, { config, user }) => {
+    const hallowRole = reaction.message.guild?.roles.cache.find(
```

Comment:
This whole action should be "skipped" if neither the `config.HALLOWEEN_ROLE` isn't defined, or the `config.HALLOWEEN_ANNOUNCEMENT` isn't defined.

Reply:
Added an initial check for those two variables, leveraging early return.